src/translate.directive.ts

```diff
@@ -27,6 +28,13 @@ export class TranslateDirective implements AfterViewChecked, OnDestroy {
                 this.checkNodes(event.translations);
             });
         }
+
+        // subscribe to onDefaultLangChange event, in case the default language changes
+        if(!this.onDefaultLangChangeSub) {
+            this.onDefaultLangChangeSub = this.translateService.onDefaultLangChange.subscribe((event: DefaultLangChangeEvent) => {
+                this.checkNodes(event.translations);
```

Comment:
Yeah there's a prob for the directive, the tests don't wait for the DOM change for some reason, it works but the test fails. It might have to do with the fact that I change the DOM manually (bad!), I should use the renderer maybe